iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/setup/ProvisioningCommon.java

```diff
@@ -427,9 +456,9 @@ else if (enrollmentType == EnrollmentType.INDIVIDUAL)
             testInstance.provisionedDeviceId = "Some-Provisioned-Device-" + testInstance.attestationType + "-" +UUID.randomUUID().toString();
             if (testInstance.attestationType == AttestationType.TPM)
             {
-                securityProvider = new SecurityProviderTPMEmulator(testInstance.registrationId);
+                securityProvider = new SecurityProviderTPMEmulator(testInstance.registrationId, MAX_TPM_CONNECT_RETRY_ATTEMPTS);
```

Comment:
create a task and reference it here as a TODO item so that we can remove the retry after e2e issue on tpm is fixed.